tracee-ebpf/tracee/tracee.go

```diff
@@ -2086,3 +2104,57 @@ func readArgFromBuff(dataBuff io.Reader) (argTag, interface{}, error) {
 	}
 	return argTag, res, nil
 }
+
+// Get information about running containers
+
+type Containers struct {
+	mapContIds map[string][]string
+}
+
+func (c *Containers) Populate() error {
+	// TODO: extend to other container engines
+	err := c.PopulateDockerContainers()
+	return err
+}
+
+func (c *Containers) Get() map[string][]string {
+	// easify range for map[string][]string
+	return c.mapContIds
+}
+
+func (c *Containers) PopulateDockerContainers() error {
+
+	// cgroupfs pids is enough to get all container tasks
+	baseDir := "/sys/fs/cgroup/pids/docker/"
```

Comment:
Hum, that is actually true. I mean, I assumed that by now all distros would still have cgroups 1 mounted and compatible but that might not be the case. This should check for both versions, and perhaps find out about the mount points from /proc/mounts. Will fix it.

Reply:
I see you said in your comment:
```
 (which might be located at /sys/fs/cgroup/unified, and not /sys/fs/cgroup that might be cgroup_v1)
```
for this, will consider.

Reply:
```
Note: this approach will not work on environments where only cgroup v1 is available, but I think we can assume that tracee will run on environments with cgroupv2 enabled. Otherwise, we will need to think of a similar approach for cgroupv1 environments.
```

I'll get the cgroups mounts from /proc/mounts, check which version OS is using and where it is mounted. Then I'll traverse v1 or v2 in a way that it finds all the pids for all existing containers.